src/pudl/metadata/resources/ferc1_eia_record_linkage.py

```diff
@@ -124,6 +127,34 @@
         "etl_group": "outputs",
         "field_namespace": "eia",
     },
+    "out_eia__yearly_assn_plant_parts_plant_gen": {
+        "description": """In order to easily determine what generator records are associated with every
+plant part record, we made this association table. This table associates every plant part
+record (identified as ``record_id_eia``) from the ``out_eia__yearly_plant_parts`` table to the associated
+'plant_gen' records (identified as ``record_id_eia_plant_gen``). All of the columns in this table
+besides ``record_id_eia`` are attributes of the plant_gen record and have a suffix of '_plant_gen'.""",
```

Comment:
I think you could add a little bit of clarity here about the different record id columns being derived from the same column in the PPE table. Also the 1:M nature of the values and the fact that some values will have the same record id in each column because some of the plant parts are single generators.

I would also say "are attributes of the `record_id_eia_plant_gen` record..." for consistency.

Reply:
i've updated the description a bunch!